ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/BlockResult.java

```diff
@@ -282,4 +296,9 @@ public String getParentBeaconBlockRoot() {
   public String getTargetBlobsPerBlock() {
     return targetBlobsPerBlock;
   }
+
+  @JsonGetter(value = "calls")
```

Comment:
"calls" does sound a bit funny for the ProcessingResults. Where does that name come from?

Reply:
from the spec :)